legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -124,6 +124,45 @@ class FilingTitles(str, Enum):
             {'types': 'BC,BEN,BC,CC,ULC', 'outputs': ['noticeOfArticles', ]},
         ]
     },
+    'amalgamation': {
+        'name': 'amalgamation',
+        'additional': [
+            {'types': 'BC,BEN,ULC,CC', 'outputs': ['noticeOfArticles', 'certificate']},
+        ],
+        'regular': {
+            'name': 'regularAmalgamation',
+            'title': 'Regular Amalgamation',
+            'displayName': 'Regular Amalgamation',
+            'codes': {
+                'BEN': 'AMARE',
+                'BC': 'AMARE',
+                'ULC': 'AMARE',
+                'CC': 'AMARE'
```

Comment:
Where did you get these filing codes?

Are you sure that we cannot use [these](https://github.com/bcgov/bcrs-shared-components/blob/2fc52fa9a8e4248a94f631909e01bb5ee0876d98/src/enums/filing-codes.ts#L10) plus the entity type?

Reply:
I think I was wrong. I updated the filing codes with [these](https://github.com/bcgov/bcrs-shared-components/blob/2fc52fa9a8e4248a94f631909e01bb5ee0876d98/src/enums/filing-codes.ts#L10)

What do you think of `temporaryCorpTypeCode`? I cannot see the `temporaryCorpTypeCode` in the FrontEnd

Reply:
What is `temporaryCorpTypeCode` needed for?

Reply:
https://github.com/bcgov/lear/pull/2336#discussion_r1408381518

It is in the `incorporationApplication`. because I think it's similar to that

Reply:
I haven't dug into `temporaryCorpTypeCode` much but I know it's used by LEAR and auth.

When Vysakh was working on registration filing, I remember he need to add a new temporary code.

It's related to bootstrapping of a business I believe.

Reply:
Oh, like TMP and RTMP?

Karim added this one in Auth: `ATMP` (A for amalg).

Reply:
oh nice!